source/access/how_to_request_more_quota.rst

```diff
@@ -87,3 +92,12 @@ maximally use. By default, this is set to 50% of the total quota for each user.
    for any user indicates what he/she can maximally use, but the actual limit
    will then depend on the usage of the other members. The total storage quota
    of the VO will always be respected.
+
+Tier-1 project
+==============
+
+The scratch storage of your project in tier-1 is managed by the institute
+hosting the tier-1 HPC infrastructure. In general, your project will get as
+much quota as requested in your application and that should be sufficient to
```

Comment:
they don't always get what they ask for ;)
```suggestion
hosting the tier-1 HPC infrastructure. In general, your project directory will get as
much quota as granted to your project and that should be sufficient to
```

Reply:
That's why I said _In general_, as in _not always_.

Reply:
that's confusing. there is no general rule, the allocation board decides how much you get.